examples/.test/trpc-tanstack-router-bug-repro/index-workaround.ts

```
@@ -0,0 +1,10 @@
+import {QueryClient} from "@tanstack/react-query";
+import type {QueryClient as TQueryClient} from "@tanstack/react-query" with {'resolution-mode': 'require'};
```

Biome (1.9.4)

[error] Import attributes cannot be used with a type-only import.

The type-only import is defined here.


(syntax/correctness/noTypeOnlyImportAttributes)